### PR TITLE
Ewl 827 table of rounds

### PR DESCRIPTION
**Pull request by narajaon**
Please check the type of changes you're about to make

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### CODE REVIEW

File: src/modules/RoundTable/RoundTable.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+const SingleDashPlaceholder = styled(Placeholder)`
+  ${({ bottom }) => bottom && 'border-bottom: none'};
+  border-right: none;
+  border-left: none;
+  border-top: none;
+`;
+
+const NeutralH5 = styled(H5)`
+  color: ${coreNeutral4};
+  ${fontAlphaHeadline};
```
Comment (JulienKode) [inline src/modules/RoundTable/RoundTable.js]: Can we use colors from theme ?

File: src/modules/RoundTable/RoundTable.js [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+    <FlexedContainer>
+      <Column>
+        {matches.map(match => (
+          <StyledScoreBlock key={match.id}>
+            <ScoreBlock.ScoreBlock matchUrl={match.matchUrl} data={match.data} hasButton={false} />
+          </StyledScoreBlock>
+        ))}
+      </Column>
+      <Column>
+        {matches.length > 1 && Array.from(Array(matches.length / 2)).map(() => <HalvedPlaceholder matches={matches} />)}
```
Comment (JulienKode) [inline src/modules/RoundTable/RoundTable.js]: Should we add test for the case where we have one match ? 🤔

File: src/modules/RoundTable/RoundTable.js [hunk 3]
```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'react-emotion';
+import { groupBy } from 'lodash';
+
+import { H5, fontAlphaHeadline } from '../../typography';
```
Comment (JulienKode) [inline src/modules/RoundTable/RoundTable.js]: Should we use absolute imports ? 

What do you think @narajaon

File: src/modules/RoundTable/RoundTable.js [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+
+const Column = styled(FlexedColum)`
+  height: 100%;
+  justify-content: space-around;
+`;
+
+const Placeholder = styled.div`
+  min-width: 30px;
+  height: 100%;
+  border: 2px solid rgba(79, 82, 106, 0.5);
```
Comment (JulienKode) [inline src/modules/RoundTable/RoundTable.js]: Can we move that color to our colors file ?

Comment (narajaon) [reply src/modules/RoundTable/RoundTable.js]: Good catch !

File: src/modules/RoundTable/RoundTable.js (hunk 3 above)
Comment (narajaon) [reply src/modules/RoundTable/RoundTable.js]: Good catch !

File: src/modules/RoundTable/RoundTable.js (hunk 1 above)
Comment (narajaon) [reply src/modules/RoundTable/RoundTable.js]: Good catch ! (i'll do another PR for all the theming / colors as soon as it's on master)

**Pull request merged**